.github/workflows/soundness.yml

```diff
@@ -0,0 +1,37 @@
+name: Soundness
```

Comment:
Can we inline this in `pull_request.yml`?

Reply:
We can but then it won't display as `PR / Soundness / Swift license headers check (pull_request)` as the other soundness and license header checks do, that's the reason for the split-out.

Reply:
What would it display as?

Reply:
I think `PR / Swift license headers check (pull_request)`

Reply:
I thought grouping the like-checks was nice. So that both license header checks appeared in the same part of the hierarchy and are grouped if you look at the `Checks` tab.